CMakeLists.txt

```diff
@@ -214,10 +214,10 @@ endif()
 
 file(GLOB_RECURSE lua_files RELATIVE "${CMAKE_HOME_DIRECTORY}/src/mudlet-lua/lua/" "${CMAKE_HOME_DIRECTORY}/src/mudlet-lua/lua/*.lua")
 list(LENGTH lua_files lua_file_count)
-if(lua_file_count EQUAL 26)
+if(lua_file_count EQUAL 25)
```

Comment:
Sorry if changing this each time the file count changes is a PITA but I put this warning in to try and prevent CMake from silently getting out of date from changes made in the qmake build system.